### PR TITLE
Change “TA” to “teacher”

### PR DESCRIPTION
**Pull request by SteveBarnett**
We don't really use "TA," so our docs shouldn't either.

### CODE REVIEW

File: sites/en/installfest/osx_panther.step [hunk 1]
```diff
@@ -41,7 +41,7 @@ For more information, see the [Apple Transition Timeline](http://en.wikipedia.or
   end
 
   option "Tiger" do
-    message "ask a TA to help you install git on Tiger"
+    message "ask a to help you install git on Tiger"
```
Comment (rorymckinley) [inline sites/en/installfest/osx_panther.step]: Missing a teacher here?

File: sites/en/workshop/teacher_training.deck.md [hunk 2]
```diff
@@ -227,15 +226,15 @@ You should be aware of these, even if you don't tell the students all of them ri
...
 <!SLIDE smbullets incremental>
-# How to effectively use TA's?
+# How to effectively use teachers?
 
 * Have them debug/look at students' computer screens
 * Pair them with the n00best/most shy students
-* Encourage TAs to answer questions
-  * TA's explanation might resonate with a student where yours didn't
+* Encourage teachers to answer questions
+  * Teachers explanation might resonate with a student where yours didn't
```
Comment (rorymckinley) [inline sites/en/workshop/teacher_training.deck.md]: Should it not be `Teacher's` rather than `Teachers` here?

**Comment by SteveBarnett** [comment]
@rorymckinley Well spotted, thank you! Changes made and pushed. Any chance of a re-review, please?

**Comment by rorymckinley** [comment]
@SteveBarnett Merged! Thanks!

**Pull request merged**